tests/functests/MultipleLogManagersTests.cpp

```diff
@@ -80,6 +102,15 @@ class MultipleLogManagersTests : public ::testing::Test,
         config1["name"] = "Instance2";
         config1["version"] = "1.0.0";
         config1["config"]["host"] = "Instance2";  // host
+
```

Comment:
Don't you need to set config2 as well?

Reply:
Good catch. Fixed - all config1, config2, and config3 are set properly now.